language/tests/unit/test_entity.py

```diff
@@ -101,6 +103,20 @@ def test_from_api_repr(self):
         for mention in entity.mentions:
             self.assertEqual(mention.mention_type, MentionType.PROPER)
 
+        # Assert that there is no sentiment.
+        self.assertIs(entity.sentiment, None)
+
+        # Assert that there *is* sentiment if it is provided.
+        payload_with_sentiment = dict(payload, sentiment={
```

Comment:
Never have I ever seen a `dict` constructed this way.

Since you never use the `payload` after this line, why not `payload['sentiment'] = ...`?